docs/data/data-grid/layout/AutoHeightGrid.tsx

```diff
@@ -15,14 +16,19 @@ export default function AutoHeightGrid() {
   });
 
   return (
-    <div style={{ width: '100%' }}>
+    <Stack spacing={1} sx={{ width: '100%', alignItems: 'flex-start' }}>
```

Comment:
One button is below the other. Why not keep them side to side like in https://mui.com/x/react-data-grid/editing/#controlled-mode?

```suggestion
    <Stack spacing={1} direction="row" sx={{ width: '100%', alignItems: 'flex-start', mb: 2 }}>
```

Reply:
I'm following up in: https://github.com/mui/mui-x/pull/5177